exercises/concept/high-school-sweetheart/lib/high_school_sweetheart.ex

```diff
@@ -12,20 +12,20 @@ defmodule HighSchoolSweetheart do
   end
 
   def pair(full_name1, full_name2) do
-    #         ******       ******
-    #       **      **   **      **
-    #     **         ** **         **
-    #    **            *            **
-    #    **                         **
-    #    **                         **
-    #     **                       **
-    #       **                   **
-    #         **               **
-    #           **           **
-    #             **       **
-    #               **   **
-    #                 ***
-    #                  *
+    #     ******       ******
+    #   **      **   **      **
+    # **         ** **         **
+    #**            *            **
+    #**                         **
+    #**     X. X.  +  X. X.     **
+    # **                       **
+    #   **                   **
+    #     **               **
+    #       **           **
+    #         **       **
+    #           **   **
+    #             ***
+    #              *
```

Comment:
Marking the position of the initials with Xs so that the student doesn't need to count spaces.

Reply:
How thoughtful ❤️